ee/ucp/admin/configure/configure-rbac-kube.md

```diff
@@ -15,47 +15,46 @@ UCP 3.0 used its own role-based access control (RBAC) for Kubernetes clusters. N
 6. Provide the YAML for the role, either by entering it in the **Object YAML** editor or select **Click to upload a .yml file** to choose and upload a .yml file instead.
-7. When you have finished specifying the YAML, Select **Create** to complete role creation.
+7. Select **Create**.
 
 ## Creating role grants
 
-Kubernetes provides 2 types of role grants:
+Kubernetes provides two types of role grants, **ClusterRoleBinding** and **RoleBinding**:
 
-- `ClusterRoleBinding` which applies to all namespaces
-- `RoleBinding` which applies to a specific namespace
+- **ClusterRoleBinding** which applies to all namespaces
+- **RoleBinding** which applies to a specific namespace
 
 To create a grant for a Kubernetes role in the UCP web interface:
 
 1. From the UCP UI, select **Access Control**.
 2. From the left navigation menu, select  **Grants**.
     ![Kubernetes Grants in UCP](/ee/ucp/images/v32grants.png)
 3. Select the **Kubernetes** tab at the top of the window. All grants to Kubernetes roles can be viewed in the Kubernetes tab.
-4. Select **Create New Grant** to start the Create Role Binding wizard and create a new grant for a given user, team or service.
+4. Select **Create New Grant** to start the Create Role Binding wizard and create a new grant for a given user, team, or service.
     ![Kubernetes Create Role Binding in UCP](../../images/kube-grant-wizard.png)
-5. Select the subject type. Your choices are:
--  **All Users**
--  **Organizations**
--  **Service account**
+5. Select the subject type, **All Users**, **Organizations**, or **Service Account**.
 6. To create a user role binding, select a username from the **Users** drop-down list then select **Next**.
 7. Select a resource set for the subject. The **default** namespace is automatically selected. To use a different namespace, select the **Select Namespace** button next to the desired namespace. For `Cluster Role Binding`, slide the **Apply Role Binding to all namespaces** selector to the right.
     ![Kubernetes Create User Role Binding in UCP](/ee/ucp/images/kube-grant-rolebinding.png)
-8. Select **Next** to continue.
-9. Select the **Cluster Role** from the drop-down list. If you create a `ClusterRoleBinding` (by selecting **Apply Role Binding to all namespaces**) then you may only select ClusterRoles. If you select a specific namespace, you can choose any role from that namespace or any ClusterRole.
+8. Select **Next**.
+9. Select the **Cluster Role** from the drop-down list. If you create a ClusterRoleBinding (by selecting **Apply Role Binding to all namespaces**), then you may only choose ClusterRoles. If you select a specific namespace, you can choose any role from that namespace or any ClusterRole.
```

Comment:
Suggest editing the second sentence to read:

"If you select **Apply Roles Binding to all namespaces** to create a ClusterRolebinding, only *ClusterRoles* can be selected. If, though, you select a specific namespace you can choose any role from that namespace or any ClusterRole."